wallet/wallet.go

```diff
@@ -3668,11 +3672,12 @@ func decodeStakePoolColdExtKey(encStr string,
 
 // Open loads an already-created wallet from the passed database and namespaces.
 func Open(db walletdb.DB, pubPass []byte, cbs *waddrmgr.OpenCallbacks,
-	voteBits uint16, stakeMiningEnabled bool, balanceToMaintain float64,
-	addressReuse bool, rollbackTest bool, pruneTickets bool, ticketAddress string,
-	ticketMaxPrice float64, ticketBuyFreq int, poolAddress string,
-	poolFees float64, ticketFee float64, addrIdxScanLen int, stakePoolColdExtKey string,
-	autoRepair, allowHighFees bool, relayFee float64, params *chaincfg.Params) (*Wallet, error) {
+	voteBits uint16, voteBitsExtended string, stakeMiningEnabled bool,
```

Comment:
I think we should be passing the new VoteBits type here, not the regular and extended vote bits separately, and especially not the extended vote bits in hex encoding.